scripts/generator/graphql_generator/csharp/SDK/ErrorFormatter.cs.erb

```diff
@@ -0,0 +1,22 @@
+namespace <%= namespace %> {
+    using <%= namespace %>.SDK;
+
+    public static class ErrorFormatter {
```

Comment:
Maybe a nice thing would be the ability to cast the `TopLevelResponse` to a `ShopifyError`. If there were no issues then `null` is returned. In this case it would be an `explicit` cast and not an `implicit` cast.